source/style-guide/writing/use-active-voice.txt

```diff
@@ -70,18 +70,18 @@ of the user from those of the technology.
 
 .. _use-passive-voice:
 
-When Do You Use Passive Voice?
-------------------------------
+When To Use Passive Voice?
```

Comment:
**issue:** These headings are all set as questions to keep the language informal. If that's not preferred, let'd do it everywhere. I would revert this change.

Reply:
How would you feel about `When Should You Use Passive Voice?`. My main complaint is that this page is an imperative "Use Active Voice" but we stop short of directly saying that there are cases when passive voice is admissible.

Reply:
**praise:** Perfect! I get the distinction and it makes more sense with `...Should You Use...`.